examples/cfd/swe_distributed_gnn/pde_dataset.py

```diff
@@ -0,0 +1,146 @@
+# coding=utf-8
```

Comment:
Need to include `# ignore_header_test` to pass the license test

Reply:
done